src/context.hpp

```diff
@@ -41,6 +41,14 @@ struct cvk_context : public _cl_context,
             }
             m_properties.push_back(*props);
         }
+        // Compute and save buffer size.
+        auto buff_size_prop_index =
+            get_property_index(CL_PRINTF_BUFFERSIZE_ARM);
+        if (buff_size_prop_index != -1) {
+            m_buffer_size = m_properties[buff_size_prop_index];
+        } else {
+            m_buffer_size = config.printf_buffer_size;
+        }
```

Comment:
technically I can default to the config.printf_buffer_size by default and get rid of this else.